src/dmd/func.d

```diff
@@ -272,6 +272,10 @@ extern (C++) class FuncDeclaration : Declaration
 
     /// local variables in this function which are referenced by nested functions
     VarDeclarations closureVars;
+
+    /// struct defining the closure memory layout
+    Symbol* closureStruct;
```

Comment:
Couldn't this information be attached to Symbol itself?

Reply:
I was following the shidden example, but would have liked to keep it in the backend, too. It seems Symbol.Sscope is unused for functions. I'll try that...

Reply:
Update amended.